meson_options.txt

```
@@ -25,6 +25,8 @@ option('tests', type : 'feature', value : 'enabled', yield : true,
   description: 'Enable or disable unit tests')
 option('introspection', type : 'feature', value : 'disabled', yield : true,
   description : 'Generate gobject-introspection bindings (.gir/.typelib files)')
+option('gtk_doc',  type : 'feature', value : 'auto', yield : true,
+  description : 'Generate API documentation with gtk-doc')
```

```suggestion
  description : 'Generate documentation with gtk-doc')
```

We use gtk-doc for the user manual as well.

---

Updated, thanks.